src/protocols/ethereum/libs/EthereumAdapter.ts

```diff
@@ -138,8 +139,9 @@ export class EthereumAdapter extends BaseProtocolAdapter {
     return isAddress(address);
   }
 
-  override async isAccountUsed(): Promise<boolean> {
-    return false; // TODO
+  override async isAccountUsed(address: string): Promise<boolean> {
+    const balance = parseFloat(await this.fetchBalance(address));
```

Comment:
We are recovering the accounts that was in used as well (with balance 0, but with history).